bool start_routing
int8 tb_id
geometry_msgs/Point tb_goal